mezo-org/mezod
# Increase gas cost of `ValidatorPool.submitApplication`

Closes: https://linear.app/thesis-co/issue/ENG-312/increase-gas-cost-of-validatorpoolsubmitapplication-by-a-constant

### Introduction

Increase the gas cost of ValidatorPool.submitApplication by a fixed factor of 4

### Changes

Changes the `SubmitApplicationMethod.RequiredGas` function to return a modified value, default * 4.

### Testing

* Run a localnet
* Perform a submitApplication transaction using hardhat
* Check gasUsed in transaction receipt

---

### Author's checklist


### Reviewer's checklist
